docker/Dockerfile

```diff
@@ -39,15 +43,17 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --defau
 ENV PATH="/root/.cargo/bin:$PATH" \
     PGX_HOME=/usr/lib/postgresql/${PG_VERSION_MAJOR}
 
-RUN cargo install --locked cargo-pgrx --version "${PGRX_VERSION}" && \
```

Comment:
Let's keep this here and just force set a new version in the pgvectorscale stage. I do agree we want to keep the same pgrx version on all of our extensions, and now that the pgrx major version was upgraded I believe it should be fairly easy to do so